app/services/encryption_service.py

```
@@ -0,0 +1,180 @@
+
+                while True:
+                    chunk = f_in.read(self.CHUNK_SIZE)
+                    if not chunk:
+                        break
+                    ciphertext = encryptor.update(chunk)
+                    f_out.write(ciphertext)
+
+                f_out.write(encryptor.finalize())
+                f_out.write(encryptor.tag)
+
+            logger.debug(f"Encrypted file: {input_path} -> {output_path}")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)